### PR TITLE
Feature/orders

### PR DESCRIPTION
**Pull request by Tkachenko01001**

### CODE REVIEW

File: src/pages/order-page/index.tsx [hunk 1]
```diff
@@ -101,6 +176,7 @@ const OrderPage = () => {
           onSubmit={handleSubmit(onSubmit)}
           noValidate
         >
+          <Box>{}</Box>
```
Comment (pqoqubbw) [inline src/pages/order-page/index.tsx]: ?

File: package.json [hunk 2]
```diff
@@ -22,8 +22,10 @@
     "@reduxjs/toolkit": "^1.9.5",
     "axios": "^1.5.0",
     "clsx": "^2.1.0",
+    "crypto-js": "^4.2.0",
     "formik": "^2.4.5",
     "js-cookie": "^3.0.5",
+    "md5": "^2.3.0",
```
Comment (pqoqubbw) [inline package.json]: for what do you need this package?

File: package.json [hunk 3]
```diff
@@ -22,8 +22,10 @@
     "@reduxjs/toolkit": "^1.9.5",
     "axios": "^1.5.0",
     "clsx": "^2.1.0",
+    "crypto-js": "^4.2.0",
```
Comment (pqoqubbw) [inline package.json:25]: same here

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: .env
Comment (pqoqubbw) [inline .env:1]: do not push this file to github, this should be local only

Comment (pqoqubbw) [inline .env:1]: do not push this `.env` to public

File: package-lock.json
Comment (pqoqubbw) [inline package-lock.json:1]: we use `yarn`

File: src/components/Layout/header/header-list-icons/headerListIcons.tsx [hunk 4]
```diff
@@ -23,7 +23,9 @@ const HeaderListIcons = () => {
   const searchToken = Object.fromEntries(searchParams.entries())
 
   useEffect(() => {
-    Object.keys(searchToken).length > 0 && setIsModalOpen(true)
+    console.log(Object.keys(searchToken))
```
Comment (pqoqubbw) [inline src/components/Layout/header/header-list-icons/headerListIcons.tsx]: remove log

File: src/components/Modal-custom-btn/ModalCustomWindow.tsx [hunk 5]
```diff
@@ -10,18 +10,21 @@ type TModalWindow = {
...
   children,
   openModal,
-  setOpenModal
+  setOpenModal,
+  callback
 }: TModalWindow) => {
   const theme = useTheme()
   const handleCloseModal = () => {
     setOpenModal(false)
+    callback && callback()
```
Comment (pqoqubbw) [inline src/components/Modal-custom-btn/ModalCustomWindow.tsx:27]:
```suggestion
    callback?.()
```

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

**Comment by Tkachenko01001** [comment]
https://app.clickup.com/t/8693kw1kr
https://app.clickup.com/t/8693kqvhx

**Pull request merged**